zio-http/src/main/scala/zhttp/http/Request.scala

```diff
@@ -19,22 +25,20 @@ trait Request extends HeaderExtension[Request] { self =>
       override def remoteAddress: Option[InetAddress] =
         self.remoteAddress
 
-      override def getBody[R, B](
-        decoder: ContentDecoder[R, Throwable, Chunk[Byte], B],
-      ): ZIO[R, Throwable, B] =
-        self.getBody(decoder)
+      override def getBody: UIO[Chunk[Byte]] =
+        UIO(d)
     }
   }
 
   /**
    * Decodes the content of the request using the provided ContentDecoder
    */
-  def getBody[R, B](decoder: ContentDecoder[R, Throwable, Chunk[Byte], B]): ZIO[R, Throwable, B]
+  def getBody: UIO[Chunk[Byte]]
```

Comment:
add `private [zhttp] getBodyAsByteBuf: UIO[ByteBuf] = ???`

Use it to implement `getBody` and `getBodyAsString`